UnitsNet.Xp.SrcGen/UnitsNet.Xp.SrcGen/QuantitySourceGenerator.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace UnitsNet.Xp.SrcGen
+{
+    [Generator]
+    public class QuantitySourceGenerator : ISourceGenerator
```

Comment:
`ISourceGenerator` has been [deprecated](https://github.com/dotnet/roslyn/blob/main/docs/features/source-generators.md) in favor of [`IIncrementalGenerator`](https://github.com/dotnet/roslyn/blob/main/docs/features/incremental-generators.md)